### DISCUSSION TITLE
How can I "Integer Scale" in Amiberry?

### DISCUSSION THREAD

**Discussion by tednilsen** [BlitterStudio/amiberry#1219] [Q&A] [answered]
In WinUAE we can enable "integer scaling" in the "filter" tab, setting the Horiz and Vert size to 1x, 1.5x, 2x and so on.
How can I do "integer scale" in Amiberry?

**Comment by midwan** [comment, marked as answer]
You can't yet, it's not implemented. :)

**Reply by tednilsen** [reply]
...did I enable it!? ;)

**Reply by midwan** [reply]
Coming back to this, Integer scaling was just added to Amiberry. You can find the Scaling option in the Display panel, under Auto/Pixelated/Smooth/Integer (choose integer here).
This works best with Auto-Crop enabled, and the new option "Res. Autoswitch" set to enabled as well, to automatically change from LowRes to HiRes and Single to Double line modes, depending on what is emulated.

This will allow games that run in say, 320x200, to be scaled up 5x in a 1080p display, without any distortion in the pixels.
